dotcom-rendering/src/web/components/Card/Card.stories.tsx

```diff
@@ -0,0 +1,382 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import { css } from '@emotion/react';
+import {
+	ArticleDesign,
+	ArticleDisplay,
+	ArticlePillar,
+	ArticleSpecial,
+} from '@guardian/libs';
+import { breakpoints, from } from '@guardian/source-foundations';
+import { storiesOf } from '@storybook/react';
+import React from 'react';
+import { Card } from './Card';
+
+const basicCardProps = {
+	linkTo: '',
+	format: {
+		display: ArticleDisplay.Standard,
+		design: ArticleDesign.Standard,
+		theme: ArticlePillar.News,
+	},
+	headlineText: 'Headline text',
+	standfirst:
+		'The 29-year-old source behind the biggest intelligence leak in the NSA’s history explains his motives',
+	headlineSize: 'medium' as SmallHeadlineSize,
+	kickerText: '',
+	webPublicationDate: '2019-11-11T09:45:30.000Z',
+	imageUrl:
+		'https://i.guim.co.uk/img/media/6537e163c9164d25ec6102641f6a04fa5ba76560/0_0_5472_3648/master/5472.jpg?width=1140&quality=85&s=15053eb16d6829d670fb348d8d26aabd',
+	imagePosition: 'top' as ImagePositionType,
+};
+
+const CardWrapper = ({ children }: { children: React.ReactNode }) => {
+	return (
+		<div
+			css={css`
+				max-height: 300px;
+				max-width: 600px;
+				flex-basis: 100%;
+				${from.tablet} {
+					flex-basis: 1;
+				}
+				margin: 10px;
+			`}
+		>
+			{children}
+		</div>
+	);
+};
+
+const CardGroup = ({ children }: { children: React.ReactNode }) => {
+	return (
+		<div
+			css={css`
+				display: flex;
+				flex-direction: column;
+				${from.tablet} {
+					flex-direction: row;
+				}
+				max-width: 1000px;
+			`}
+		>
+			{children}
+		</div>
+	);
+};
+
+const CardsWithDifferentThemes = ({
+	display,
+	design,
+	title,
+}: {
+	display: ArticleDisplay;
+	design: ArticleDesign;
+	title: string;
+}) => {
+	const cards = [];
+	for (const [pillarName, pillarValue] of Object.entries(ArticlePillar)) {
+		if (Number.isNaN(Number(pillarName))) {
```

Comment:
Would be nice to have a comment to explain that this is a [workaround the TS enum gotcha][ts], where the object is basically mirrored between keys and values:

```ts
console.log(Object.entries(ArticlePillar));
/*
[
    ["0", "News"],
    ["1", "Opinion"],
    ["2", "Sport"],
    ["3", "Culture"],
    ["4", "Lifestyle"],
    ["News", 0],
    ["Opinion", 1],
    ["Sport", 2],
    ["Culture", 3],
    ["Lifestyle",4]
] 
*/
```

[ts]: https://www.typescriptlang.org/play?#code/KYOwrgtgBAggTgFwJYGMA2wAKS1oIZxQDeAsAFBSVQBywA7gM5QC8UADADTlVQDyADkhBIA9iBZQAjFwpUAyvxGIJAJhk8AwmDQIwcYBIDM6qgBkkAM2AMEATwwSALDIC+5cijEMRGAHRoRAHMACl4AIwArYBQEX1AEOCRrYPhkdCwcfDgASmyAbnIAegAqcgBtbioygCI2ao4oatpGaoBdE0oayXrGgSFREDaOqBqVHuqFJQQhys7qw3GtHT1gGdk5x3HzKxt7VfbZkab6Bh62A-WjvuExHskLnhrJxB6VB6rqpd19HsN3ue21jsGHqjla5FaUHIxUKQA

Reply:
👍 done - thanks for the playground link!